widgets/src/App/App.js

```diff
@@ -0,0 +1,287 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import {
+  AppBar,
+  Checkbox,
+  Container,
+  FormControlLabel,
+  Toolbar,
+  Typography,
+  IconButton,
+} from '@material-ui/core';
+import { Menu as MenuIcon } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+
+import { authenticate } from 'api/app-builder/pages';
+import WIDGETS_CONFIG from 'mocks/app-builder/widgets';
+import SETTINGS from 'mocks/app-builder/pages';
+import Menu from 'app/Menu';
```

Comment:
```
Casing of app/Menu does not match the underlying filesystem
```

it should be `'App/Menu'`

Reply:
it's the only capitalized folder in that level, so it seemed out of place

Reply:
I agree, but the folder is still called `App`, can we rename it to `app`?

Reply:
seems like folder name change did not commit...